.github/workflows/build-live-branch.yml

```diff
@@ -31,6 +31,12 @@ jobs:
             - name: Setup WooCommerce Monorepo
               uses: ./.github/actions/setup-woocommerce-monorepo
 
+            - name: Cache PNPM Dependencies
+              uses: actions/setup-node@8f152de45cc393bb48ce5d89d36b731f54556e65
+              with:
+                node-version-file: .nvmrc
+                cache: pnpm
+
```

Comment:
Again, solid improvement here.  Down to 6 minutes from 13 or 14 (it looks like).

Reply:
Oh, I wish it was that good =) I observed around 10 seconds of improvements, but we should get more consistent with build times (the time needed to download all packages might differ).

IIRC we have some stats related to builds, hope we can pull some numbers next week and see the impact of the changes.